server/statistics/store_collection_test.go

```diff
@@ -27,8 +27,10 @@ var _ = Suite(&testStoreStatisticsSuite{})
 type testStoreStatisticsSuite struct{}
 
 func (t *testStoreStatisticsSuite) TestStoreStatistics(c *C) {
-	opt := mockoption.NewScheduleOptions()
-	opt.LocationLabels = []string{"zone", "host"}
+	opt := config.NewTestOptions()
+	rep := opt.GetReplicationConfig().Clone()
+	rep.LocationLabels = []string{"zone", "host"}
+	opt.SetReplicationConfig(rep)
```

Comment:
Maybe it can also be encapsulated?

Reply:
There is no cluster here. (and cannot create a cluster either because of dependency matters)